gapis/resolve/dependencygraph2/graph_visualization/graph_visualization.go

```diff
@@ -49,16 +50,23 @@ func createGraphFromDependencyGraph(dependencyGraph dependencygraph2.DependencyG
 						idToBuilder[command.API().ID()] = graphVisualizationAPI.GetGraphVisualizationBuilder()
 					}
 					builder := idToBuilder[command.API().ID()]
+					commandNameAndNodeId := fmt.Sprintf("%s_%d", command.CmdName(), cmdNodeId)
+					isSubCommand, parentNodeId := false, 0
 
 					label, name := "", ""
 					if len(cmdNode.Index) == 1 {
 						label = builder.GetCommandLabel(command, cmdNodeId)
 						name = command.CmdName()
+						commandNameAndNodeIdToNodeId[commandNameAndNodeId] = int(nodeId)
```

Comment:
The naming here is kind of confusing. In "commandNameAndNodeIdToNodeId", the first "NodeId" seems to refer to the CmdID (`cmdNode.Index[0]`), while the second seems to refer to the `nodeId dependencygraph2.NodeID`. If I'm understanding this correctly, would "cmdNameAndIdToNodeId" be correct? 

"commandNameAndNodeId" should probably also be changed to match.

Reply:
Yes, you are right. I changed it, in addition to this I renamed 'cmdNodeId' to 'cmdId'. I think now it is more clear.